build/webpack/webpack.extension.node.config.js

```diff
@@ -57,9 +58,20 @@ const config = {
                 test: /\.ts$/,
                 exclude: /node_modules/,
                 use: [
-                    {
-                        loader: 'ts-loader'
-                    }
+                    fasterCompiler
+                        ? // Esbuild doesn't have type checking, hence we need to not use esbuild
+                          // in some cases (so we catch issues with the type checker).
+                          {
+                              loader: 'esbuild-loader',
+                              options: {
+                                  loader: 'ts',
+                                  target: ['es6', 'es2018', 'ES2019', 'ES2020'],
```

Comment:
Copied from the tsconfig file
Unfortunately esbuild requires us to manually provide these